EndlessClient/Controllers/ArrowKeyController.cs

```diff
@@ -128,6 +128,8 @@ private void AttemptToStartWalking()
                     _unwalkableTileActionsHandler.HandleUnwalkableTileActions(unwalkableActions, cellState);
                     break;
                 case WalkValidationResult.GhostComplete:
+                    _characterAnimationActions.StartWalking(Option.None<MapCoordinate>(), true);
```

Comment:
Can you add the explicit parameter notation here, so it's more obvious what the bool is? `, ghosted: true` instead of just `, true`

Reply:
@ethanmoffat fixed!